src/plugins/pcr_primer_design_for_dna_assembly/src/PCRPrimerDesignForDNAAssemblyOPWidget.cpp

```diff
@@ -274,7 +274,10 @@ void PCRPrimerDesignForDNAAssemblyOPWidget::sl_selectManually() {
     auto sequenceSelection = annDnaView->getActiveSequenceContext()->getSequenceSelection();
     int startRegionValue = sbStartRegion->value();
     int endRegionValue = sbEndRegion->value();
-    sequenceSelection->setSelectedRegions({ U2Region(startRegionValue, endRegionValue - startRegionValue) });
+    if (startRegionValue > 0) {
+        sequenceSelection->setSelectedRegions({U2Region(static_cast<qint64>(startRegionValue) - 1,
```

Comment:
nit: You do not need these ugly cast when you move from int32 to int64

Reply:
Without cast there is the warning C26451 "Arithmetic overflow: Using operator '-' on a 4 byte value and then casting the result to a 8 byte value. Cast the value to the wider type before calling operator '-' to avoid overflow (io.2)".